Makefile

```diff
@@ -152,7 +152,9 @@ functions-examples-all:
 	done
 
 test-go-mod:
-	./hack/for-each-module.sh "go list -m -json all > /dev/null && go mod tidy -v"
+	./hack/for-each-module.sh $$(pwd)/hack/replace.sh; \
+	./hack/for-each-module.sh "go mod tidy -v"; \
+	./hack/for-each-module.sh $$(pwd)/hack/dropReplace.sh
```

Comment:
I tried inlining the scripts in `replace.sh` and `dropReplace.sh`, but couldn't figure out how to escape the relevant characters.